sources/packages/api/src-sql/StudentsAddGenderAndDOB/Add-GenderAndDOB.sql

```diff
@@ -0,0 +1,24 @@
+-- Adding new column date_of_birth
+-- Add the column with a dummy default value that will be removed in the next command.
+ALTER TABLE sims.students
+    ADD COLUMN birth_date Date NOT NULL DEFAULT now();
+
+-- Removing the dummy constraint created just to allow us
+-- to add a "NOT NULL" column in an exising table.
+ALTER TABLE sims.students
```

Comment:
For safety purpose  Can we add IF EXISTS / IF NOT EXISTS clause.

Reply:
Added the "IF EXISTS / IF NOT" and tested the scripts executing more than one time in the db.